File: 프로그래머스/0/181904. 세로 읽기/세로 읽기.kt
class Solution {
    fun solution(my_string: String, m: Int, c: Int): String {
        var answer = StringBuilder()
        val rows = my_string.length / m

        for (i in 0 until rows) {
            val index = i * m + (c - 1)
            answer.append(my_string[index])
        }

        return answer.toString()
    }
}

File: 프로그래머스/0/181904. 세로 읽기/세로 읽기.js
function solution(my_string, m, c) {
    let answer = '';
    // 전체 줄 수는 my_string 길이 / m
    for (let i = 0; i < my_string.length / m; i++) {
        // i번째 줄에서 c번째 열의 문자는 인덱스 (i * m + (c - 1))
        answer += my_string[i * m + (c - 1)];
    }
    return answer;
}
